python/lib/dependabot/python/python_versions.rb

```diff
@@ -4,12 +4,13 @@ module Dependabot
   module Python
     module PythonVersions
       PRE_INSTALLED_PYTHON_VERSIONS = %w(
-        3.10.7
+        3.11.0
       ).freeze
 
       # Due to an OpenSSL issue we can only install the following versions in
       # the Dependabot container.
       SUPPORTED_VERSIONS = %w(
+        3.11.0
         3.10.7 3.10.6 3.10.5 3.10.4 3.10.3 3.10.2 3.10.1 3.10.0
```

Comment:
Note to myself:
Looks like a few more python patch versions were released, need to add those too from https://www.python.org/downloads/

Reply:
https://github.com/dependabot/dependabot-core/pull/6043